src/store.ts

```diff
@@ -45,8 +45,8 @@ type SubscribeInvalidateTuple<T> = [Subscriber<T>, Invalidater<T>];
 
 /**
  * Creates a `Readable` store that allows reading by subscription.
- * @param value initial value
- * @param start start and stop notifications for subscriptions
+ * @param [value] initial value
```

Comment:
I think this is required, no? A readable store without the second argument doesn't do anything, and since the second argument is required the first one is too

```suggestion
 * @param value initial value
```